@here/harp-omv-datasource/test/OmvTest.ts

```diff
@@ -26,6 +26,16 @@ import { assert } from "chai";
  */
 
 export class RoadFilter implements OmvFeatureFilter {
+    hasKindFilter: boolean = true;
+
+    wantsKind(kind: string | string[]): boolean {
+        return (
+            kind === undefined ||
+            (typeof kind === "string" && kind.includes("road")) ||
+            (Array.isArray(kind) && (kind as string[]).filter(s => s.includes("road")).length > 0)
```

Comment:
Why not use array.includes so we can break once we find one match?

Reply:
It is just test code... As long as it works and is readable, I am willing to live with it.

Reply:
Ok, I overlooked this was test code